### PR TITLE
Fix/heroku media

### PR DESCRIPTION
**Pull request by NewIvanIvanov**

### CODE REVIEW

File: backend/models/issues.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 import os
 
-from flask import current_app
+from flask import current_app, send_from_directory
```
Comment (serge30) [inline backend/models/issues.py]: Is `send_from_directory` used anywhere here?

File: backend/models/issues.py [hunk 2]
```diff
@@ -36,9 +36,16 @@ def delete(self):
         if not os.listdir(directory_path):
             os.rmdir(directory_path)
 
+    def get_full_thumbnail_url(self):
+        url = self.get_thumbnail_url()
+        if current_app.config.get('MEDIA_URL'):
+            return '%s%s' % (current_app.config['MEDIA_URL'], url)
+        return '%s%s' % ('/media/', url)
```
Comment (serge30) [inline backend/models/issues.py]: It doesn't make sense to use formatting here, concatenation is enough:
```
       if current_app.config.get('MEDIA_URL'):
            return current_app.config['MEDIA_URL'] + url
       return '/media/' + url
```

File: client/apps/city_issues/migrations/0001_initial.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```
Comment (serge30) [inline client/apps/city_issues/migrations/0001_initial.py]: How this migration is related to media URLs handling?

**Review by serge30 — CHANGES_REQUESTED** [review]

File: backend/models/issues.py (hunk 1 above)
Comment (NewIvanIvanov) [reply backend/models/issues.py]: no, i"ll fix it

File: client/apps/city_issues/migrations/0001_initial.py (hunk 3 above)
Comment (NewIvanIvanov) [reply client/apps/city_issues/migrations/0001_initial.py]: fixed

File: backend/models/issues.py (hunk 2 above)
Comment (NewIvanIvanov) [reply backend/models/issues.py]: fixed

**Review by serge30 — APPROVED** [review]

**Review by andrewRv — APPROVED** [review]

**Review by forever-Agriculture — APPROVED** [review]

**Pull request merged**